VKappProject/VKappProject/Controllers/AuthorizationViewController.swift

```diff
@@ -0,0 +1,109 @@
+// AuthorizationViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import UIKit
+import WebKit
+
+/// Экран авторизации
+final class AuthorizationViewController: UIViewController {
+    // MARK: - Private IBOutlet
+
+    @IBOutlet private var webView: WKWebView! {
+        didSet {
+            webView.navigationDelegate = self
+        }
+    }
+
+    // MARK: - Private Properties
+
+    private let session = Session.shared
```

Comment:
```suggestion
    private let session = Session.shared
```
это в сетевом слое должно быть